examples/react/react-threejs/src/gameComponents/Players.tsx

```diff
@@ -0,0 +1,50 @@
+import { useDojo } from "@/dojo/useDojo";
+import { Has, defineSystem } from "@dojoengine/recs";
+import { Player } from "./Player";
+import { useEffect, useState } from "react";
+
+export const Players = (props: any) => {
+    const {
+        setup: {
+            clientComponents: { Position },
+            world,
+        },
+    } = useDojo();
+
+    const [players, setPlayers] = useState<any>({});
```

Comment:
it makes sense we convert this to zustand state for local storage, then these players can be accessed else where in the app